src/groupeddataframe/groupeddataframe.jl

```diff
@@ -328,6 +330,8 @@ Base.convert(::Type{Array{T}}, key::GroupKey) where {T} = Vector{T}(key)
 Base.Array(key::GroupKey) = Vector(key)
 Base.Array{T}(key::GroupKey) where {T} = Vector{T}(key)
 
+Base.Dict(key::GroupKey) = Dict(Symbol(first(ki)) => last(ki) for ki in pairs(key))
```

Comment:
This should be enough I think as `keys` will produce names as `Symbol` already (maybe this will resolve the CI errors?)
```suggestion
Base.Dict(key::GroupKey) = Dict(pairs(key))
```